metals/src/main/scala/scala/meta/internal/metals/MetalsEnrichments.scala

```diff
@@ -361,4 +361,19 @@ object MetalsEnrichments extends DecorateAsJava with DecorateAsScala {
 
   }
 
+  implicit class XtensionDocumentSymbol(documentSymbol: l.DocumentSymbol) {
+    def toSymbolOccurrence: s.SymbolOccurrence = s.SymbolOccurrence(
+      range = Some(
+        new s.Range(
+          documentSymbol.getRange.getStart.getLine,
+          documentSymbol.getRange.getStart.getCharacter,
+          documentSymbol.getRange.getStart.getLine,
+          documentSymbol.getRange.getStart.getCharacter
+        )
+      ),
+      symbol = documentSymbol.getName,
```

Comment:
Include `range.getEnd.getLine`

Reply:
this way we test the end line by printing the number in the output expect file, while discarding the end offset for the symbol occurrence range.